backend/experiment/rules/congosamediff.py

```diff
@@ -0,0 +1,142 @@
+
+from django.utils.translation import gettext_lazy as _
+from experiment.actions.final import Final
+from experiment.models import Experiment
+from section.models import Playlist as PlaylistModel
+from session.models import Session
+from experiment.actions import ChoiceQuestion, Explainer, Form, Playlist, Trial
+from experiment.actions.playback import Autoplay
+from .base import Base
+from result.utils import prepare_result
+
+
+class CongoSameDiff(Base):
+    ''' A micro-PROMS inspired experiment that tests the participant's ability to distinguish between different sounds. '''
+    ID = 'CONGOSAMEDIFF'
+    contact_email = 'aml.tunetwins@gmail.com'
+
+    def __init__(self):
+        pass
+
+    def first_round(self, experiment: Experiment):
+        ''' Provide the first rounds of the experiment,
+        before session creation
+        The first_round must return at least one Info or Explainer action
+        Consent and Playlist are often desired, but optional
+        '''
+
+        # 1. Playlist
+        playlist = Playlist(experiment.playlists.all())
+
+        # 2. Explainer
+        explainer = Explainer(
+            instruction='Welcome to this Same Diff experiment',
+            steps=[],
+            step_numbers=True
```

Comment:
You can leave `step_numbers` out here, as you don't provide any steps.

Reply:
```suggestion
```